var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -164,7 +164,6 @@ def patch(self):
     @property
     def common_config_args(self):
         config_args = [
-            '-prefix', self.prefix,
```

Comment:
Does `qt` require `-prefix` or `--prefix`? If it actually requires the latter, then it's probably best to leave things as they were before so that we override `configure` completely. If you check in `lib/spack/spack/build_systems/autotools.py`, you'll see that `configure` basically calls `configure_args` and appends `--prefix`. But if we actually need `-prefix`, then `configure_args` won't help.